src/components/ui/Modal/index.tsx

```diff
@@ -24,22 +24,25 @@ export interface ModalProps
   size?: ModalSize;
   /** The rootId of the modal. */
   rootId?: string;
+  /** Optional prop to prevent the modal from closing. */
+  dismissible?: boolean;
 }
 
 export const Modal: FC<ModalProps> = ({
   size = 'md',
   onClose,
   children,
   rootId,
+  dismissible = false,
```

Comment:
Wouldn't we want `dismissable` to be true to maintain default behavior?

Reply:
Yep, I think I got everything turned around at the end of last week. Updating now.